### PR TITLE
Write test for Command

### PR DESCRIPTION
**Pull request by vladis84**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  | comma-separated list of tickets # fixed by the PR, if any

In Symfony command property  $defaultName and $defaultDescription is deprecated. Сan i fix it?

### CODE REVIEW

File: tests/Command/Schema/SchemaClearBaseCommandTest.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        $schemaProvider->expects($this->once())->method('clear');
+        $container = new SimpleContainer([SchemaProviderInterface::class => $schemaProvider]);
+        $promise = new CycleDependencyProxy($container);
+        $command = new SchemaClearCommand($promise);
+        $input = $this->getMockBuilder(InputInterface::class)->getMock();
+        $output = $this->getMockBuilder(OutputInterface::class)->getMock();
+
+        $code = $command->run($input, $output);
+
+        $this->assertEquals(ExitCode::OK, $code);
```
Comment (samdark) [inline tests/Command/Schema/SchemaClearBaseCommandTest.php:29]: Should we check that schema was actually cleaned up?

**Comment by samdark** [comment]
> In Symfony command property $defaultName and $defaultDescription is deprecated. Сan i fix it?

By using attribute? Yes but please do it separately.

File: tests/Command/Schema/SchemaClearBaseCommandTest.php (hunk 1 above)
Comment (vladis84) [reply tests/Command/Schema/SchemaClearBaseCommandTest.php:30]: Method return code OK. Verify run method clear. ($schemaProvider->expects($this->once())->method('clear');)

File: tests/Command/Schema/SchemaClearBaseCommandTest.php [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Yiisoft\Test\Support\Container\SimpleContainer;
+use Yiisoft\Yii\Console\ExitCode;
+use Yiisoft\Yii\Cycle\Command\CycleDependencyProxy;
+use Yiisoft\Yii\Cycle\Command\Schema\SchemaClearCommand;
+use Yiisoft\Yii\Cycle\Schema\SchemaProviderInterface;
+
+class SchemaClearBaseCommandTest extends TestCase
```
Comment (vjik) [inline tests/Command/Schema/SchemaClearBaseCommandTest.php:16]:
```suggestion
final class SchemaClearBaseCommandTest extends TestCase
```

File: tests/Command/Schema/SchemaCommandTest.php [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Output\Output;
+use Symfony\Component\Console\Output\OutputInterface;
+use Yiisoft\Test\Support\Container\SimpleContainer;
+use Yiisoft\Yii\Console\ExitCode;
+use Yiisoft\Yii\Cycle\Command\CycleDependencyProxy;
+use Yiisoft\Yii\Cycle\Command\Schema\SchemaCommand;
+
+class SchemaCommandTest extends TestCase
```
Comment (vjik) [inline tests/Command/Schema/SchemaCommandTest.php:17]:
```suggestion
final class SchemaCommandTest extends TestCase
```

File: tests/Command/Schema/SchemaRebuildCommandTest.php [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Output\BufferedOutput;
+use Yiisoft\Test\Support\Container\SimpleContainer;
+use Yiisoft\Yii\Console\ExitCode;
+use Yiisoft\Yii\Cycle\Command\CycleDependencyProxy;
+use Yiisoft\Yii\Cycle\Command\Schema\SchemaRebuildCommand;
+use Yiisoft\Yii\Cycle\Schema\SchemaProviderInterface;
+
+class SchemaRebuildCommandTest extends TestCase
```
Comment (vjik) [inline tests/Command/Schema/SchemaRebuildCommandTest.php:14]:
```suggestion
final class SchemaRebuildCommandTest extends TestCase
```

**Review by vjik — APPROVED** [review]

**Comment by samdark** [comment]
Thank you!

**Pull request merged**